packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.50.2"
+  changes:
+    - description: Add EMR logs dashboard.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/6895
```

```suggestion
      link: https://github.com/elastic/integrations/pull/7075
```
and should this be 1.50.1 instead?